src/backtick/core.clj

```diff
@@ -28,40 +34,37 @@
   []
   (engine/cron-map))
 
-(defn perform
+(defn schedule
   "Run a job on the backtick queue"
-  [name & args]
-  (if (not (@engine/workers name))
-      (log/errorf "No worker %s registered, not queuing job" name)
-      (engine/add name args)))
+  [registered-function & args]
+  (let [name (@engine/workers registered-function)]
+    (if (not name)
+        (log/errorf "No worker %s registered, not queuing job" registered-function)
```

Comment:
It may be more informative to say something like: "Worker % not registered. Register this function before calling `schedule`." That gives the user a path to a solution rather than just telling him he failed.